charts/federation-v2/values.yaml

```diff
@@ -10,7 +10,7 @@ controllermanager:
   replicaCount: 1
   repository: quay.io/kubernetes-multicluster
   image: federation-v2
-  tag: canary
+  tag: v0.0.7
```

Comment:
Please remove this commit - the image tag should remain canary in master.  Only the archived chart should have its tag set, and the tag should reflect the release the chart is intended to be a part of.

Reply:
Removing.